src/index.js

```diff
@@ -1,34 +1,37 @@
 const path = require('path')
 const fs = require('fs-extra')
 const { defaultsDeep } = require('lodash')
-const { execSync } = require('child_process')
+const chokidar = require('chokidar');
 
 const {
   parseHosts,
   writeExtensionTemplates,
   enablePlayerDebugMode,
   symlinkExtension,
+  copyDependencies,
+  copyIcons,
 } = require('./utils')
 
-const { createManifest } = require('./manifest')
-
 module.exports = async bundler => {
-  bundler.on('bundled', async bundle => {
-    await createManifest({ bundle })
```

Comment:
We can use this plugin to get the manifest: https://www.npmjs.com/package/parcel-plugin-bundle-manifest

Also, we don't need the manifest to figure out the filename of the ExtendScript bundle since that is bundled in another parcel process as the entry file and thus won't get a weird filename